airflow/gcp/hooks/tasks.py

```diff
@@ -68,14 +70,14 @@ def get_conn(self):
     @GoogleCloudBaseHook.fallback_to_default_project_id
     def create_queue(
         self,
-        location,
-        task_queue,
-        project_id=None,
-        queue_name=None,
-        retry=None,
-        timeout=None,
-        metadata=None,
-    ):
+        location: str,
+        task_queue: Union[Dict, Queue],
+        project_id: Optional[str] = None,
+        queue_name: Optional[str] = None,
+        retry: Optional[Retry] = None,
+        timeout: Optional[float] = None,
+        metadata: Optional[Sequence[Tuple[str, str]]] = None
+    ) -> Queue:
```

Comment:
Can you add return type in other places?

Reply:
Return types added as well.